services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/logging/PpacErrorCode.java

```diff
@@ -5,11 +5,11 @@
 
 public enum PpacErrorCode {
   // iOS related error codes
-  API_TOKEN_ALREADY_ISSUED(SecurityLogger::warn),
+  API_TOKEN_ALREADY_ISSUED(SecurityLogger::securityWarn),
```

Comment:
Every error code is related to access control then why don't we just use the security logging for all of them:
API_TOKEN_ALREADY_ISSUED
API_TOKEN_EXPIRED
API_TOKEN_QUOTA_EXCEEDED
DEVICE_BLOCKED
DEVICE_TOKEN_INVALID
DEVICE_TOKEN_REDEEMED
DEVICE_TOKEN_SYNTAX_ERROR

Reply:
agreed with security team and @mlenkeit to always log those events with same 'log-settings'